### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by bwee** [jbohnslav/deepethogram#8]
In the feature extraction step, after ~100 epochs, my validation loss begins increasing while my train loss continues to decrease. Do you have any suggestions on which hyperparameters to tweak to better generalize the model and prevent overfitting to the training set?

**Comment by jbohnslav** [comment]
How strong is the overfitting? A few % is to be expected. Dropout is already set very high, around 0.9 by default. By default, the learning rate should decrease when validation loss plateaus, and then the model should stop itself.
